contrib/job-scripts/coreudmp-handling/post.d/002-link-core-to-tests

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+import errno
+import glob
+import os
+import re
+import sys
+
+if 'AVOCADO_JOB_LOGDIR' not in os.environ:
+    sys.exit(1)
+
+AVOCADO_JOB_LOGDIR = os.environ['AVOCADO_JOB_LOGDIR']
+if not os.path.isdir(AVOCADO_JOB_LOGDIR):
+    sys.exit(2)
+
+COREDUMPS_DIR = os.path.join(AVOCADO_JOB_LOGDIR, 'coredumps')
+if not os.path.isdir(COREDUMPS_DIR):
+    sys.exit(0)
+
+COREDUMPS = glob.glob(os.path.join(COREDUMPS_DIR, 'core.*'))
+if not COREDUMPS:
+    sys.exit(0)
+
+# If we reached this far, there are core dumps, so let's attempt to
+# link them to the test results directories.
+#
+# This pattern list can be expanded, and if a match occurs, it should
+# return as the first group member the PID (an unsigned integer) of
+# the process which may have an associated coredump file.
+PATTERNS = [r'.*\sPID\s(\d+)']
+
+regexes = map(re.compile, PATTERNS)
```

Comment:
Top level variables should use UPPER_CASE_CHARS. (but better change would be to implement `__main__`).